### PR TITLE
feat: Added support for maxPayload, raftHeartbeatTimeout, raftElectionTimeout, raftLeaseTimeout, raftCommitTimeout native nats settings

### PR DESCRIPTION
**Pull request by daniel-codefresh**
Checklist:

* [x] My organization is added to [USERS.md](https://github.com/argoproj/argo-events/blob/master/USERS.md).

### CODE REVIEW

**Comment by daniel-codefresh** [comment]
Related issue: #1309

File: controllers/eventbus/installer/nats.go [hunk 1]
```diff
@@ -483,14 +520,18 @@ streaming {
 	node_id: $POD_NAME
 	peers: [%s]
 	log_path: /data/stan/logs
+	raft_heartbeat_timeout: "%s"
+    raft_election_timeout: "%s"
```
Comment (whynowy) [inline controllers/eventbus/installer/nats.go:524]: minor - indent

Comment (daniel-codefresh) [reply controllers/eventbus/installer/nats.go]: weird.. it looked perfectly indented on GoLand
Anyway, I've fixed it, thanks

**Review by whynowy — APPROVED** [review]
LGTM!

**Comment by daniel-codefresh** [comment]
@whynowy any chance we can add this commit to v1.5.7?

**Comment by whynowy** [comment]
I think we should release v1.6.

**Comment by daniel-codefresh** [comment]
> I think we should release v1.6.

@whynowy Yes that totally makes sense, but do we have any ETA for the release of v1.6? Or are we waiting for any additional features?

**Comment by whynowy** [comment]
I really want to get https://github.com/argoproj/argo-events/issues/1459 in v1.6, it should be ready in 1 week or so.

**Comment by daniel-codefresh** [comment]
> I really want to get #1459 in v1.6, it should be ready in 1 week or so.

Gotcha👍

**Pull request merged**